### PR TITLE
CA-335606: Add a ping step to wait until network ready

### PR DESCRIPTION
**Pull request by xihuan-citrix**

### CODE REVIEW

**Comment by DeliZhangX** [comment]
@xihuan-citrix need to fix Travis scan issue

**Comment by xihuan-citrix** [comment]
> @xihuan-citrix need to fix Travis scan issue
fixed

File: autocertkit/network_tests.py [hunk 1]
```diff
@@ -515,6 +515,9 @@ def _run_test(self, session, mode):
 
         log.debug("Starting test...")
         results = []
+        # First time try ping to make sure network stable
+        ping(vm1_ip, vm2_bondnic_ip, vm2_bondnic_dev)
```
Comment (acefei) [inline autocertkit/network_tests.py]: There is a duplicate code, you might bind a variable to the return of ping, and then just results.append(var)

Comment (xihuan-citrix) [reply autocertkit/network_tests.py]: It is not a duplicate code, it is just a step to try to ping first but not add into results, you can see later ping serveral times.
```
# First time try ping to make sure network stable
        ping(vm1_ip, vm2_bondnic_ip, vm2_bondnic_dev)

        # Test healthy bond
        results.append(ping(vm1_ip, vm2_bondnic_ip, vm2_bondnic_dev))

        # Test degraded bond
        set_nic_device_status(session, self.piface, 'down')
        results.append(ping(vm1_ip, vm2_bondnic_ip, vm2_bondnic_dev))

        # Test degraded bond
        set_nic_device_status(session, self.piface, 'up')
        set_nic_device_status(session, self.siface, 'down')
        results.append(ping(vm1_ip, vm2_bondnic_ip, vm2_bondnic_dev))
```

**Pull request merged**